x/auction/simulation/params.go

```diff
@@ -17,29 +17,29 @@ func ParamChanges(r *rand.Rand) []simulation.ParamChange {
 	// as strings in JSON (such as time.Duration) have the escaped quotes.
 	// TODO should we encode the values properly with ModuleCdc.MustMarshalJSON()?
 	return []simulation.ParamChange{
-		simulation.NewSimParamChange(types.ModuleName, string(types.KeyBidDuration), "",
+		simulation.NewSimParamChange(types.ModuleName, string(types.KeyBidDuration),
 			func(r *rand.Rand) string {
-				return fmt.Sprintf("\"%d\"", GenBidDuration(r))
+				return fmt.Sprintf("%d", GenBidDuration(r))
```

Comment:
Should these have escaped quotes?
My understandig was that param changes are stored as json encoded param values. ie time.Duration encodes to a string.

Reply:
good point. Idk tbh, maybe it's the same? The SDK uses both from what I'm seeing

Reply:
After some digging, I think we should do this:
```suggestion
				return string(types.ModuleCdc.MustMarshalJSON(GenBidDuration(r)))
```
When these generated values (string types) are set in the params keeper (through the param gov handler) they're type cast to `[]byte` then unmarshalled from json (using amino) ,  then validated, then marshalled back to json bytes and then set in the param store.

So I think the string formatting is effectively marshalling the types to json, which sounds like a bad idea. I propose replacing it with the proper json marshaller.

I looked at v0.38 sdk modules. Whether they use quotes or not mirrors amino's decision to use quotes when marshalling to json. Which itself seems a bit arbitrary: uint16 - no quotes, uint64 - quotes. It's also inconsistent with golang's encoding/json library (eg marshalling to json with encoding/json then unmarhaling with amino can produce incorrect values)

I'll create an issue